_posts/2015-02-04-2.markdown

```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+title:  "Taking a Photo"
+date:   2015-02-04 10:00:02
+answerrostag: "simulation_challenge"
+track: [simulation]
+---
+
+[comment]: <> (TODO: replace answerrostag and track)
```

Comment:
A new track would be great. Go ahead and pick something. We can always change it later.